### DISCUSSION TITLE
Cloudflare 520 error

### DISCUSSION THREAD

**Discussion by guiqis** [chatwoot/chatwoot#8654] [Self hosted]
### Describe the bug

Hi guys,

I'm facing a problem with my chatwoot installation that I have no idea how to solve. Only in safari, in the regular tab (anonymous the problem does not occur), I'm seeing a 520 error. In chrome or any other browsers it works fine. But in safari, it does not - and only when I'm logged in.

Does anyone has ever faced this problem and have any idea of how to solve it?

### To Reproduce

Reach the chatwoot installation URL when logged in

### Environment

Linux VM

### Cloud Provider

DigitalOcean

### Platform

Browser

### Operating system

MacOS 14.2.1

### Browser and version

Safari 17.2.1

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-835/cloudflare-520-error">PR-835 Cloudflare 520 error</a></p>

**Comment by pranavrajs** [comment]
Moving to self-hosted as it is not related to Chatwoot. This is [what I can find](https://developers.cloudflare.com/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-5xx-errors/#error-520-web-server-returns-an-unknown-error), however none of this seems specific to a browser.

**Comment by richardfeynman1** [comment]
Possible Issue:
Check cookie size stores: chrome allows max 4096 size
I think Chatwoot stores different cookies for different types of logins..eg. normal user, superuser, etc.
And if you try to login for different roles in the same browser. This issue crops up.

There are some other possible scenarios as well.
Happy to help if the above one does not work.
